drivers/vulkan/rendering_device_vulkan.cpp

```diff
@@ -1569,16 +1569,18 @@ RID RenderingDeviceVulkan::texture_create(const TextureFormat &p_format, const T
 #warning TODO check for support via RenderingDevice to enable on mobile when possible
 #endif
 	// vkCreateImage fails with format list on Android (VK_ERROR_OUT_OF_HOST_MEMORY)
-#ifndef ANDROID_ENABLED
+	VkImageFormatListCreateInfoKHR format_list_create_info; //keep out of the if, needed for creation
```

Comment:
This never gets used on Android so it is resulting in a compiler warning and travis is failing